sdk/core/azure-core-experimental/pom.xml

```diff
@@ -72,16 +72,15 @@
   <dependencies>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-core</artifactId>
-      <version>1.38.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
+      <artifactId>azure-json</artifactId>
```

Comment:
We can drop this dependency as azure-core brings this transitively.

Reply:
I prefer having libraries directly including dependencies used directly. Helps with transitive dependencies being overridden based on Maven dependency resolution.